src/test/erasure-code/ceph_erasure_code_benchmark.cc

```diff
@@ -121,9 +128,19 @@ int ErasureCodeBench::setup(int argc, char** argv) {
     exhaustive_erasures = false;
   if (vm.count("erased") > 0)
     erased = vm["erased"].as<vector<int> >();
-
-  k = stoi(profile["k"]);
-  m = stoi(profile["m"]);
+	
+  try {
+    k = stoi(profile["k"]);
+    m = stoi(profile["m"]);
+  } 
+  catch (const std::invalid_argument& e) {
```

Comment:
might want to move this line up into the previous one to be consistent with the coding style of the source tree.

probably we could just catch `std::logic_error` and print out `e.what()` instead of both `invalid_argument` and `out_of_range`?

Reply:
catch `std::logic_error` is good, and it may be helpful to  tell that k & m is the point because e.what( ) only prints `stoi`, which is not enough for some users . I'll then modify the patch.

Reply:
sorry , it seems I cannot pass the `Signed-off-by` test because I didn't add `Signed-off-by:` info in the first commit [8d2c593](https://github.com/ceph/ceph/pull/29864/commits/8d2c593a08147f480c37b229a8ec572e7503842d) even if I update the patch, is there anyway I can fix this?